recipes/spritz/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.3.3" %}
+
+package:
+  name: spritz
+  version: {{ version }}
+
+source:
+  url: https://github.com/smith-chem-wisc/Spritz/releases/download/{{ version }}/SpritzCMD.zip
+  sha256: f218e3ebb599b29960ae0245aa4a16ab8734ed7895d98f0772b1f7734dacfc10
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - sysroot_linux-64 2.17  # [linux64]
```

Comment:
why is this needed?

Reply:
These are prerequisites accompanying dotnet's need for CentOS 7.

See:
https://conda-forge.org/docs/maintainer/knowledge_base.html#using-centos-7
https://github.com/conda-forge/dotnet-feedstock/blob/master/conda-forge.yml#L5
https://github.com/conda-forge/dotnet-feedstock/blob/master/recipe/meta.yaml#L47

Do you think I should remove this, since CentOS 7 is the default container for bioconda now?

Reply:
I'll give this recipe a try with these removed.